environment/templates/fabric8-tenant-che.yml

```diff
@@ -3,10 +3,10 @@ apiVersion: v1
 kind: Template
 metadata:
   labels:
-    app: fabric8-tenant-che
+    app: fabric8-tenant-che-mt
     provider: fabric8
     version: ${COMMIT}
-  name: fabric8-tenant-che
+  name: fabric8-tenant-che-mt
```

Comment:
not sure if it makes sense to rename the file since in metadata `fabric8-tenant-che-mt` is still used all over the places

Reply:
this is something that I wasn't sure about either. My thinking was that having only `fabric8-tenant-che-mt.yaml` file with the `mt` suffix could be confusing - that there is the normal one (without suffix) missing. But today morning, looking at the templates - this new state is maybe more confusing :-D.
To be honest, it's up to you - you are the main person who is working with the templates, so I'll name it by whatever you want to :-)

Reply:
ok, let's leave it as `fabric8-tenant-che-mt` for now, we could later remove *-mt (multi-tenant) prefix and update template metadata.